editor/src/clj/editor/breakpoints_view.clj

```diff
@@ -0,0 +1,526 @@
+    :icon "icons/32/Icons_S_14_linkarrow.png"
+    :command :breakpoints.show-in-assets}
+   {:label (localization/message "breakpoints.context-menu.jump-to-line")
+    :command :breakpoints.jump-to-line}
+   menu-items/separator
+   {:label (localization/message "breakpoints.context-menu.edit-selected")
+    :command :breakpoints.edit-selected}
+   {:label (localization/message "breakpoints.context-menu.toggle-selected-enabled")
+    :command :breakpoints.toggle-selected-enabled}
+   menu-items/separator
+   {:label (localization/message "breakpoints.context-menu.remove-selected")
+    :command :breakpoints.remove-selected}])
+
+(defn- breakpoints-toolbar-view [project localization-state state swap-state]
+  (let [breakpoints (:breakpoints state)
+        action-all-fn (fn [action-fn _]
+                        (g/with-auto-evaluation-context evaluation-context
+                          (set-regions-with-action! swap-state project evaluation-context breakpoints breakpoints action-fn)))]
+    {:fx/type fx.h-box/lifecycle
+     :id "breakpoints-tool-bar"
+     :anchor-pane/top 0
+     :anchor-pane/left 0
+     :anchor-pane/right 0
+     :pref-height toolbar-height
+     :alignment :center-left
+     :spacing 10
+     :children [{:fx/type fx.button/lifecycle
+                 :id "breakpoints-enable-all"
+                 :text (localization-state (localization/message "breakpoints.button.enable-all"))
+                 :on-action (fn/partial action-all-fn #(update-breakpoints-enabled-state %1 %2 true))}
+                {:fx/type fx.button/lifecycle
+                 :id "breakpoints-disable-all"
+                 :text (localization-state (localization/message "breakpoints.button.disable-all"))
+                 :on-action (fn/partial action-all-fn #(update-breakpoints-enabled-state %1 %2 false))}
+                {:fx/type fx.button/lifecycle
+                 :id "breakpoints-toggle-all"
+                 :text (localization-state (localization/message "breakpoints.button.toggle-all"))
+                 :on-action (fn/partial action-all-fn toggle-breakpoints-enabled)}
+                {:fx/type fx.button/lifecycle
+                 :id "breakpoints-remove-all"
+                 :text (localization-state (localization/message "breakpoints.button.remove-all"))
+                 :on-action (fn/partial action-all-fn #(vec (remove (set %2) %1)))}]}))
+
+(defn- icon-button [icon-path scale classes on-action-event]
+  {:fx/type fx.button/lifecycle
+   :style-class (into ["icon-button"] classes)
+   :graphic {:fx/type fx.stack-pane/lifecycle
+             :min-width 20
+             :min-height 20
+             :alignment :center
+             :children [{:fx/type fx.svg-path/lifecycle
+                         :content icon-path
+                         :scale-x scale
+                         :scale-y scale
+                         :style-class ["icon-button-graphic"]}]}
+   :on-action on-action-event})
```

Comment:
This is fine, but note we also have a `fxui/icon-graphic` function that scales an SVG path to fit within a specified pixel size bounds if it would help here. It requires the SVG path to be added to the `fxui/icon-svg-path-data` map, though!